src/main/java/exchange/core2/core/orderbook/IOrderBook.java

```diff
@@ -203,6 +211,10 @@ static CommandResultCode processCommand(final IOrderBook orderBook, final OrderC
             cmd.marketData = orderBook.getL2MarketDataSnapshot(size >= 0 ? size : Integer.MAX_VALUE);
             return CommandResultCode.SUCCESS;
 
+        } else if (commandType == OrderCommandType.SUSPEND_USER) {
+
+            return orderBook.cancelOrdersByUid(cmd);
```

Comment:
Suspending users should not cancel existing orders as this is expensive (not low-latency) operation. Approach is different. I will put a description into the PR.

Reply:
Check this idea please:
https://github.com/mzheravin/exchange-core/pull/102#issuecomment-859148950